doc/menu_lua_api.txt

```diff
@@ -266,6 +267,7 @@ Package - content which is downloadable from the content db, may or may not be i
         {
             name             = "name of content",
```

Comment:
this should be removed then?

Reply:
Content still has a name, it's the technical identify. For mods and txps, the `name`. For games, it's the `id`

Reply:
the example should be `name = "technical_id"` then

Reply:
```suggestion
            name             = "technical_id",
```